shell/packages/accounts-identity/accounts-identity-client.js

```diff
@@ -30,14 +30,61 @@ Accounts.isLinkingNewIdentity = function () {
 };
 
 Template.identityLoginInterstitial.onCreated(function () {
-  this._state = new ReactiveVar({ justLoggingIn: true });
+  // An exhaustive list of the top-level states this interstitial may be in.
+  // Each state may include an object with additional context-specific information.
+  //
+  //   * loading                 Waiting on subscriptions to finish loading
+  //   * creatingAccount         We have sent a createAccountForIdentity call and are awaiting the reply
+  //   * accountCreationFailed   The createAccountForIdentity call failed.  Value is the Error.
+  //   * loggingInWithIdentity   We have sent a loginWithIdentity call and are awaiting the reply
+  //   * loginWithIdentityFailed The loginWithIdentity call failed.  Value is the Error.
+  //   * loggingInWithToken      We have started logging in with a token and are awaiting the reply
+  //   * loginWithTokenFailed    The loginWithToken call failed.  Value is the Error.
+  //   * linkingIdentity         We have sent a linkIdentityToAccount call and are awaiting the reply
+  //   * noLoginIdentities       Subscriptions loaded, but we can neither login, create, nor link an account.  We require input.
```

Comment:
I might find that a bit ambiguous with `loginWithIdentityFailed`.

Maybe `identityIsNotLoginIdentity`?